chart/mits/assets/run.sh.tmpl

```diff
@@ -0,0 +1,46 @@
+{{- /*
+   Copyright 2020 SUSE
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/ -}}
+
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+cf api --skip-ssl-validation {{ .Values.cf.api | squote }}
+{{/* cf auth uses the CF_USERNAME and CF_PASSWORD environment variables. They are set in the mits job. */}}
```

Comment:
I would have expected a `#` comment, since that would be nice in the generated script too.

No, I'm not asking you to change it, that's too much :D